crates/component-macro/src/bindgen.rs

```diff
@@ -170,11 +170,16 @@ impl Parse for Config {
                 path = Some(input.parse::<syn::LitStr>()?.value());
             }
         }
-        let (resolve, pkg, files) = parse_source(&path, &inline, &features)
+        let (resolve, pkgs, files) = parse_source(&path, &inline, &features)
             .map_err(|err| Error::new(call_site, format!("{err:?}")))?;
 
         let world = resolve
-            .select_world(pkg, world.as_deref())
+            .select_world(
+                *pkgs
+                    .get(0)
+                    .ok_or_else(|| Error::new(call_site, "at least one package must be defined"))?,
+                world.as_deref(),
```

Comment:
With https://github.com/bytecodealliance/wasm-tools/pull/1611 this'll ferry along the `pkgs` argument to `select_world` which is what we want here.